pkg/util/util.go

```diff
@@ -1432,3 +1432,11 @@ func GetCommandStringFromEnvs(envVars []v1alpha2.EnvVar) string {
 	}
 	return setEnvVariable
 }
+
+func GetEnvWithDefault(key string, defaultval string) string {
```

Comment:
Can you add some comments to this function? Also, it's being used in tests script, IIUC. Why not put it in `helper` package under `tests`?

Reply:
Updated